### PR TITLE
feat(styles): horizon button delta

### PR DESCRIPTION
**Pull request by mikerodonnell89**

### CODE REVIEW

File: src/styles/avatar.scss [hunk 1]
```diff
@@ -23,22 +23,22 @@ $block: #{$fd-namespace}-avatar;
   );
```
Comment (platon-rov) [inline src/styles/avatar.scss:23]: Please remove `tabindex`

Screenshot 2021-11-04 at 12 39 46

File: src/styles/button.scss [hunk 2]
```diff
@@ -60,6 +60,7 @@ $block: #{$fd-namespace}-button;
   line-height: 1rem;
```
Comment (platon-rov) [inline src/styles/button.scss:60]: Looks broken 

Screenshot 2021-11-04 at 12 41 44

File: src/styles/button.scss [hunk 3]
```diff
@@ -60,6 +60,7 @@ $block: #{$fd-namespace}-button;
...
   text-align: center;
```
Comment (platon-rov) [inline src/styles/button.scss:61]: When pressing the button it gets the appropriate border color. And just after it gets the blue border, is it intended?

https://user-images.githubusercontent.com/20265336/140300705-12f23de8-e684-49a9-8110-0951c938ef1d.mov

File: src/styles/card.scss [hunk 4]
```diff
@@ -16,7 +16,6 @@ $fd-card-sections-horizontal-spacing: 2rem !default;
 $fd-card-sections-vertical-spacing: 1.75rem !default;
```
Comment (platon-rov) [inline src/styles/card.scss:16]: Screenshot 2021-11-04 at 12 48 31

File: src/styles/message-strip.scss [hunk 5]
```diff
@@ -18,7 +18,7 @@ $block: #{$fd-namespace}-message-strip;
   @mixin fd-message-strip-icon-container {
     position: absolute;
     width: 2.5rem;
-    top: 0.4375rem;
+    top: 0.3435rem;
```
Comment (platon-rov) [inline src/styles/message-strip.scss:21]: `line-height: 2rem` may be used to avoid such magical value.

File: src/styles/panel.scss [hunk 6]
```diff
@@ -18,6 +18,9 @@ $block: #{$fd-namespace}-panel;
     @include fd-reset();
   }
 
+  background-color: var(--fdPanel_Background);
+  border-radius: var(--fdPanel_Border_Radius);
+
   &__header {
```
Comment (platon-rov) [inline src/styles/panel.scss:24]: Should we have rounded `border-bottom` for collapsed panels?

Screenshot 2021-11-04 at 12 55 01

**Review by platon-rov — COMMENTED** [review]
Great work! 

Please describe changes in pr.

File: src/styles/button.scss (hunk 3 above)
Comment (mikerodonnell89) [reply src/styles/button.scss:61]: Yes, the blue border is the focus outline

File: src/styles/button.scss (hunk 2 above)
Comment (mikerodonnell89) [reply src/styles/button.scss:60]: Will be implementing the segmented button in my next PR

File: src/styles/avatar.scss (hunk 1 above)
Comment (mikerodonnell89) [reply src/styles/avatar.scss:23]: Unrelated to this PR

**Comment by mikerodonnell89** [comment]
@platon-rov I think there was some confusion as I had mistakenly initially opened this PR to be merged into `main` rather than `feature/horizon-delta`

**Comment by mikerodonnell89** [comment]
Closing because I will have 1 PR for button and segmented button

**Pull request closed without merging**